app/controllers/account/dashboard_controller.rb

```diff
@@ -3,5 +3,18 @@ class Account::DashboardController < ApplicationController
 
   def index
     @chart_value = current_companies
+    @count_of_pages = todays_pages
+  end
+
+  private
+
+  def todays_pages
+    count = 0
+    current_companies.each do |company|
+      count += company.pages
+                      .where('created_at > ?', 1.day.ago)
```

Comment:
PLUS 

I expect this code to be a part of new class

CompanyPagesAnalytics.count_pages to return it